great-idea/css/index.css

```diff
@@ -63,4 +58,177 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	width: 876px;
```

Comment:
I would recommend against using the body as your container. changing the width will leave the html dom element exposed and this can cause some issues. I suggest that you just create a container div that wraps around all of your elements and just use that to change your widths. You can still use the body as a root for all the styles you want to pass down like colors and font sizes but I would suggest against using it as a wrapper.